modules/packages/Image.chpl

```diff
@@ -155,22 +157,21 @@ module Image {
     outfile.writeBinary(0:uint(32), endianness.little); /* "important" colors */
 
     for i in d.dim(0) {
-      var nbits = 0;
       for j in d.dim(1) {
         var p = pixels[i,j];
-        var redv = (p >> colorOffset(red)) & colorMask;
-        var greenv = (p >> colorOffset(green)) & colorMask;
-        var bluev = (p >> colorOffset(blue)) & colorMask;
+        var redv = (p >> colorOffset(rgbColor.red)) & colorMask;
+        var greenv = (p >> colorOffset(rgbColor.green)) & colorMask;
+        var bluev = (p >> colorOffset(rgbColor.blue)) & colorMask;
```

Comment:
if you `use rgbColor` you wouldn't need to qualify

Reply:
I prefer the qualified access, there aren't that many uses of the colors and it frees of "simple" identifiers to be used as variable names.